pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/LetterTemplate.kt

```diff
@@ -78,34 +78,38 @@ sealed class Element<out Lang : LanguageSupport> {
     }
 
     data class Table<Lang : LanguageSupport>(
-        val title: List<Element<Lang>>?,
         val rows: List<Row<Lang>>,
-        val columnHeaders: List<Row<Lang>>,
+        val columnHeader: Header<Lang>,
     ) : Element<Lang>() {
-        val width: Int
 
         init {
-            val cellWidths = rows.map { it.cells.sumOf { cell -> cell.cellColumns } }.distinct()
-            if (cellWidths.size > 1) {
-                throw IllegalArgumentException("rows in the table needs to have the same number of columns")
```

Comment:
Kan du lage en ny type Exception for disse feilene? InvalidTableDeclarationException eller noe sånt?

Reply:
Done.